tools/src/main/scala/org/apache/spark/sql/rapids/tool/AppFilterImpl.scala

```diff
@@ -87,8 +92,26 @@ class AppFilterImpl(
     } else {
       apps
     }
+
+   // If `any` is specified as logicFilter config, then store the value in variable which is used
+   // later for final computation.
+   val appNameFilteredForDisjunction = if (appArgs.applicationName.isSupplied &&
+       filterAppName.nonEmpty && appArgs.any()
+       && logicFilter.contains("any")) {
```

Comment:
What condition would lead to `appArgs.any()` being true but `logicFilter` not being set to "any"?

Reply:
It's redundant. Removed `logicFilter`.